File: CS/WindowsApplication3/MVVM/Data/Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MVVMSample.MVVM.Data {
    public class Order {
        [Required]
        public int ID { get; set; }
        public DateTime OrderDate { get; set; }
        public decimal Freight { get; set; }
    }
}

File: VB/WindowsApplication3/MVVM/Data/Order.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace MVVMSample.MVVM.Data
    Public Class Order
        <Required> _
        Public Property ID() As Integer
        Public Property OrderDate() As Date
        Public Property Freight() As Decimal
    End Class
End Namespace
